api/nodes/serializers.py

```diff
@@ -273,6 +278,30 @@ class NodeDetailSerializer(NodeSerializer):
     id = IDField(source='_id', required=True)
 
 
+class NodeForksSerializer(NodeSerializer):
+
+    category_choices = Node.CATEGORY_MAP.items()
+    category_choices_string = ', '.join(["'{}'".format(choice[0]) for choice in category_choices])
+
+    title = ser.CharField(required=False)
+    category = ser.ChoiceField(read_only=True, choices=category_choices, help_text="Choices: " + category_choices_string)
+    forked_date = ser.DateTimeField(read_only=True)
+
+    def create(self, validated_data):
+        node = validated_data.pop('node')
```

Comment:
Yes, I was trying to more-or-less match front-end functionality.  Actually when creating a fork, you can't even change the title, it is automatically 'Fork of + ' current node title.  But the method for creating a fork already had the title as an argument so I thought forking would be improved by allowing you to change the title on creation.  But to mach the front end completely, no fields change on creation.